units/XNA0105/XNA0105_script.lua

```diff
@@ -67,6 +77,8 @@ XNA0105 = Class(NAirUnit) {
         end
     end,
 
+    ---@param self XNA0105
+    ---@param large any
```

Comment:
This seems like a boolean.
```suggestion
    ---@param large boolean
```